common/scala/src/main/scala/whisk/core/entity/AuthKey.scala

```diff
@@ -16,6 +16,13 @@
 
 package whisk.core.entity
 
+import scala.util.Try
+
+import spray.json.JsString
+import spray.json.JsValue
+import spray.json.RootJsonFormat
+import spray.json.deserializationError
```

Comment:
You can ignore these comments if you want. But I'ld strongly recommend to configure your IDE to leave Wildcard imports alone and always use the wildcard ones, at least for spray.json. It makes later editing of these classes and files a whole lot easier. Won't block the PR though, just a thing i noticed.